tests/test_cookiecutter_xblock.py

```diff
@@ -78,15 +76,11 @@ def test_bake_selecting_license(cookies, license_name, target_string, custom_tem
 
 def test_readme(options_baked, custom_template):
     """The generated README.rst file should pass some sanity checks and validate as a PyPI long description."""
-    readme_file = Path('README.rst')
-    readme_lines = [x.strip() for x in readme_file.open()]
+    readme_lines = [rl.strip() for rl in Path('README.rst').read_text().splitlines()]
     assert "My First XBlock" == readme_lines[0]
     assert 'Testing with Docker' in readme_lines
-    try:
-        os.system("python -m build --wheel")
-        os.system("twine check dist/*")
-    except os.ErrorReturnCode as exc:
-        pytest.fail(str(exc))
+    sh.python("-m", "build", "--wheel")
+    sh.twine("check", "dist/*")
```

Comment:
I'm surprised this actually works! Twine must be doing glob-expansion on its arguments.

Reply:
`sh` passes the strings through to the program.  Twine probably does glob expansion to support Windows.
```
% cat argv.py
import sys
print(sys.argv)

% python -c "import sh; print(sh.python('argv.py', 'foo/*'))"
['argv.py', 'foo/*']
```